uthash.yaml

```
@@ -0,0 +1,33 @@
+package:
+  name: uthash
+  version: "2.3.0"
+  epoch: 0
+  description: "Header file for providing hash table for C structures"
+  copyright:
+    - license: "BSD-2-Clause"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - wolfi-baselayout
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/troydhanson/uthash.git
+      tag: v${{package.version}}
+      expected-commit: e493aa90a2833b4655927598f169c31cfcdf7861
+
+  - runs: mkdir -p ${{targets.destdir}}/usr/include
+
+  - runs: cp src/uthash.h ${{targets.destdir}}/usr/include/
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: troydhanson/uthash
+    strip-prefix: v
```

The upstream repo doesn't publish releases, only tags - so we need to specify to poll tags here:

```suggestion
    strip-prefix: v
    use-tag: true
```